tests/unit/selectmenu/selectmenu.html

```diff
@@ -12,7 +12,7 @@
 	<script src="../../../external/qunit-assert-classes/qunit-assert-classes.js"></script>
 	<script>
 	TestHelpers.loadResources({
-		css: [ "core", "menu" , "selectmenu" ],
+		css: [ "core", "menu" , "selectmenu", "button" ],
```

Comment:
This needs to be reflected in the [`css.structure` comments](https://github.com/arschmitz/jquery-ui/blob/button-fixup/ui/selectmenu.js#L15-L16).